### PR TITLE
Added Glassmorphism design generating web tool

### PR DESCRIPTION
**Pull request by SushilBajracharya01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -617,6 +617,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [DopeUI](https://dopeui.co/): High quality website UI templates
 - [Coolors](https://coolors.co/): The super fast color schemes generator
 - [uiGradients](https://uigradients.com/): An awesome gradient generator
+- [glassmorphism generator](https://ui.glass/generator/): Amazing tool to generate glassmorphism design.
```
Comment (shahednasser) [inline README.md:620]: Can you capitalize the first letter of each word?

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @SushilBajracharya01 that's a nice one! I just have a small comment below

File: README.md (hunk 1 above)
Comment (SushilBajracharya01) [reply README.md:620]: :) Thank you, yes I will do it right away

**Comment by shahednasser** [comment]
Great thanks!

**Pull request merged**